seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-1/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcMysqlIT.java

```diff
@@ -242,61 +343,116 @@ Pair<String[], List<SeaTunnelRow>> initTestData() {
         for (int i = 0; i < 100; i++) {
             byte byteArr = Integer.valueOf(i).byteValue();
-            SeaTunnelRow row =
-                    new SeaTunnelRow(
-                            new Object[] {
-                                i % 2 == 0 ? (byte) 1 : (byte) 0,
-                                new byte[] {byteArr},
-                                new byte[] {byteArr, byteArr},
-                                new byte[] {byteArr, byteArr, byteArr, byteArr},
-                                new byte[] {
-                                    byteArr, byteArr, byteArr, byteArr, byteArr, byteArr, byteArr,
-                                    byteArr
-                                },
-                                i % 2 == 0 ? Boolean.TRUE : Boolean.FALSE,
-                                i,
-                                i,
-                                i,
-                                i,
-                                i,
-                                i,
-                                i,
-                                i,
-                                i,
-                                Long.parseLong("1"),
-                                Long.parseLong("1"),
-                                Long.parseLong("1"),
-                                BigDecimal.valueOf(i, 0),
-                                BigDecimal.valueOf(i, 18),
-                                BigDecimal.valueOf(i, 18),
-                                Float.parseFloat("1.1"),
-                                Float.parseFloat("1.1"),
-                                Double.parseDouble("1.1"),
-                                Double.parseDouble("1.1"),
-                                "f",
-                                String.format("f1_%s", i),
-                                String.format("f1_%s", i),
-                                String.format("f1_%s", i),
-                                String.format("f1_%s", i),
-                                String.format("{\"aa\":\"bb_%s\"}", i),
-                                String.format("f1_%s", i),
-                                Date.valueOf(LocalDate.now()),
-                                Timestamp.valueOf(LocalDateTime.now()),
-                                Time.valueOf(LocalTime.now()),
-                                new Timestamp(System.currentTimeMillis()),
-                                "test".getBytes(),
-                                "test".getBytes(),
-                                "test".getBytes(),
-                                "test".getBytes(),
-                                "test".getBytes(),
-                                "f".getBytes(),
-                                bigintValue.add(BigDecimal.valueOf(i)),
-                                decimalValue.add(BigDecimal.valueOf(i)),
-                                decimalValue.add(BigDecimal.valueOf(i)),
-                            });
+            SeaTunnelRow row;
+            if (i == 99) {
```

Comment:
Maybe should remove the if else and add null row outside the loop.

Reply:
At the beginning, I also placed it outside and kept reporting errors before placing it inside the loop.

Reply:
I know why this phenomenon happened

I use the 'getJdbcCase()' method to obtain the 'JdbcCase' class

And this method is the factory method.